### PR TITLE
DOC-4383 added Boost.redis to community client table

### PR DESCRIPTION
**Pull request by andy-stark-redis**
[DOC-4383](https://redislabs.atlassian.net/browse/DOC-4383)

[DOC-4383]: https://redislabs.atlassian.net/browse/DOC-4383?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/develop/connect/clients/_index.md
Comment (dwdougherty) [inline content/develop/connect/clients/_index.md:42]: The README is titled "PhpRedis". Doesn't have to match, so I'll leave it to you.

```suggestion
| [PHP](https://www.php.net/) | PhpRedis extension | https://github.com/phpredis/phpredis | https://github.com/phpredis/phpredis/blob/develop/README.md |
```

File: content/develop/connect/clients/_index.md
Comment (dwdougherty) [inline content/develop/connect/clients/_index.md:46]: Similar comment to the last.

```suggestion
| [C++](https://en.wikipedia.org/wiki/C%2B%2B) | Boost.Redis | https://github.com/boostorg/redis | https://www.boost.org/doc/libs/develop/libs/redis/doc/html/index.html |
```

**Review by dwdougherty — APPROVED** [review]
Two things for you to consider. Approving so I'm not holding you up.

**Comment by andy-stark-redis** [comment]
@dwdougherty Good suggestions, thanks!

**Pull request merged**
